packages/host/app/components/operator-mode/code-submode/spec-preview.gts

```diff
@@ -600,49 +602,42 @@ export default class SpecPreview extends GlimmerComponent<Signature> {
   };
 
   <template>
-    <PrerenderedCardSearch
-      @query={{this.specQuery}}
-      @format='fitted'
-      @realms={{this.realms}}
-    >
-      <:response as |cards|>
-        {{#let (this.getSpecIntent cards) as |showCreateSpecIntent|}}
-          {{yield
-            (component
-              SpecPreviewTitle
-              showCreateSpecIntent=showCreateSpecIntent
-              createSpec=this.createSpec
-              isCreateSpecInstanceRunning=this.createSpecInstance.isRunning
-              specType=this.specType
-              numberOfInstances=cards.length
-            )
-            (component
-              SpecPreviewContent
-              showCreateSpecIntent=showCreateSpecIntent
-              canWrite=this.canWrite
-              onSelectCard=this.onSelectCard
-              selectedId=this._selectedCardId
-              spec=this.card
-              isLoading=false
-              cards=cards
-            )
-          }}
-        {{/let}}
-      </:response>
-      <:loading>
```

Comment:
The pre-rendered search api is not javascript driven. So, occasionally we wud have workarounds that made is bad. Particularly, assigning cardIds within the `:response` block. This caused more re-renders